test/aircraft/FlightManagementSystem/WaypointModel.spec.js

```diff
@@ -26,6 +30,34 @@ ava('#hold returns properties for a holding pattern', (t) => {
     t.true(model.hold.timer === -999);
 });
 
+ava('#isVector returns true when name starts with `#`', (t) => {
+    const model = new WaypointModel(vectorWaypointPropsMock);
+
+    t.true(model.isVector);
+});
+
+ava('#isVector returns false when name does not contain `#`', (t) => {
+    const props = Object.assign({}, waypointMock, { legLength: '3min', turnDirection: 'right' });
+    const model = new WaypointModel(props);
+
+    t.false(model.isVector);
+});
+
+ava('#vector returns undefined when #isVector is false', (t) => {
+    const props = Object.assign({}, waypointMock, { legLength: '3min', turnDirection: 'right' });
```

Comment:
this is a nitpick but, did you mean to use a hold waypoint here?  this could likely just be the `waypointMock` no?